sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -98,14 +121,116 @@
 		return service.getDeletedUuids(since, user);
 	}
 
+	public Map<Disease, List<FeatureConfigurationIndexDto>> getEnabledFeatureConfigurations(FeatureConfigurationCriteria criteria) {
+		List<FeatureConfigurationIndexDto> featureConfigurations = getFeatureConfigurations(criteria, false);
+		Map<Disease, List<FeatureConfigurationIndexDto>> diseaseListMap = new TreeMap<>();
+		featureConfigurations.forEach(featureConfigurationIndexDto -> {
+			if (!diseaseListMap.containsKey(featureConfigurationIndexDto.getDisease())) {
+				diseaseListMap.put(featureConfigurationIndexDto.getDisease(), new ArrayList<>());
+			}
+			diseaseListMap.get(featureConfigurationIndexDto.getDisease()).add(featureConfigurationIndexDto);
+		});
+		return diseaseListMap;
+	}
+
+	public Page<FeatureConfigurationIndexDto> getIndexPage(
+		FeatureConfigurationCriteria criteria,
+		Integer offset,
+		Integer size,
+		List<SortProperty> sortProperties) {
+		List<FeatureConfigurationIndexDto> featureConfigurationIndexList = getIndexList(criteria, offset, size, sortProperties);
+		CountryReferenceDto serverCountry = countryFacadeEjb.getServerCountry();
+		long totalElementCount = count(new DistrictCriteria().country(serverCountry).region(criteria.getRegion()));
+		return new Page<>(featureConfigurationIndexList, offset, size, totalElementCount);
+	}
+
+	private List<FeatureConfigurationIndexDto> getIndexList(
+		FeatureConfigurationCriteria criteria,
```

Comment:
As we discussed, make sure that you only query for line listing configurations here, i.e. specify the feature type in the criteria and add it to the filter